src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Figure;
+import core.basesyntax.suppliers.FigureSupplier;
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        Figure[] figures = new Figure[new Random().nextInt(6)];
```

Comment:
1. save random in the variable like `figureSupplier`
2. `6` is a magic number, use constants instead